packages/client/src/views/CorrectionForm/VerifyCorrector.tsx

```diff
@@ -109,15 +113,20 @@ class VerifyCorrectorComponent extends React.Component<IFullProps> {
   }
 
   getGenericCorrectorInfo = (corrector: string): ICorrectorInfo => {
-    const { intl, declaration, form } = this.props
+    const { intl, declaration, form, registerForm } = this.props
     const info = declaration.data[corrector]
     //TODO :: we have to get form defination from new certificateCorrectorDefination
     const showInfoFor = ['mother', 'father', 'child', 'informant']
+
+    const eventRegistrationInput = draftToGqlTransformer(
+      registerForm,
+      declaration.data
+    )
```

Comment:
We could tighten up the type of this eventRegistrationInput for strictening the types